hw1/02-palindrome.js

```diff
@@ -1,3 +1,49 @@
-const elem = document.querySelector('input');
+//citations: w3schools, CSS airbnb style guide
 
-elem.addEventListener('input', handleInput);
+const elem = document.querySelector("input");
+const output = document.getElementById("output");
+
+elem.addEventListener("input", handleInput);
+
+function is_palindrome(num) {
+	const myArr = String(num)
+		.split("")
+		.map((num) => {
+			return Number(num);
+		});
+	//clone the array and reverse it
+	const new_arr = [...myArr].reverse();
+	if (myArr.toString() === new_arr.toString()) {
+		return true;
+	} else {
+		return false;
+	}
```

Comment:
This could be simplified to just return the result of the comparison instead

Reply:
Thanks Tim, 
 I have made the changes accordingly